cli/commands/clean.js

```diff
@@ -18,13 +15,11 @@ class CleanCommand {
           process.exit(1)
         }
         ProcessManager.disconnect()
-        if (existsSync(path.join(homedir(), '.rainbow', 'nearup', 'main.py'))) {
-          try {
-            console.log('Stopping nearup')
-            execSync('python3 ~/.rainbow/nearup/main.py stop')
-          } catch (err) {
-            console.log(`Error stopping nearup ${err}`)
-          }
+        try {
+          console.log('Stopping nearup')
+          execSync('nearup stop')
```

Comment:
The only problem I see here it's not that you add this to the ci:
```
export PATH=$HOME/.local/bin:$PATH
```
the same problem will happen to normal bridge users, as if the don't have this in the path, the nearup command will fail as it is not in the in the path, I think you will have to use the full path for that reason.

Reply:
Regular bridge users can install it through pip3 as usual added note about in the README
They still require install of Rust to run anything, don't see as big issue installing nearup through pip then

For regular users docker should be used, which is unfortunately not working at the moment

Reply:
Done, other way around, waiting for test to pass, looks ok now

Reply:
@chefsale @mhalambek this is not working for me locally since nearup is not being installed in this location `$HOME/.local/bin`. If we are installing `nearup` from pip I think it is safe to assume that nearup will be in the path.

Reply:
Of you run the prepare it should be installed in that folder. Have you initialize it with that?